ui/jobs/jobs.js

```diff
@@ -565,6 +569,18 @@ class Bars {
       this.o.healthBar.fg = computeBackgroundColorFrom(this.o.healthBar, 'hp-color');
   }
 
+  _updateProcBoxFlashState() {
+    if (this.options.FlashExpiredProcsInCombat) {
+      const boxes = document.getElementsByClassName('proc-box');
+      for (const box of boxes) {
+        if (this.inCombat)
+          box.setAttribute('incombat', '');
+        else
+          box.removeAttribute('incombat');
```

Comment:
Maybe a CSS `class` is more suitable for this usage?

Reply:
I don't mind either, will change